src/main/resources/assets/carpet/lang/zh_cn.json

```diff
@@ -1,12 +1,208 @@
+  "rule.hopperCounters.extra.4": "物品会被销毁，每个漏斗每tick销毁一组物品",
 
-  "rule.portalSuffocationFix.desc": "使下界传送门正确地传送实体",
+  "rule.huskSpawningInTemples.name": "尸壳在神殿里生成",
+  "rule.huskSpawningInTemples.desc": "只有尸壳会在沙漠神殿生成",
+
+  "rule.lagFreeSpawning.name": "延迟自然生成",
+  "rule.lagFreeSpawning.desc": "生物生成需要更少的CPU和内存",
+
+  "rule.leadFix.name": "栓绳修复",
+  "rule.leadFix.desc": "修复栓绳在未加载区块中，变得不可见的问题",
+
+  "rule.maxEntityCollisions.name": "最大实体挤压数",
+  "rule.maxEntityCollisions.desc": "设定最大实体挤压限制，0表示无限大",
+
+  "rule.mergeTNT.name": "合并TNT",
+  "rule.mergeTNT.desc": "合并静止的点燃的tnt实体",
 
+  "rule.missingTools.name": "工具缺失修复",
+  "rule.missingTools.desc": "活塞，玻璃和海绵可以用合适的工具更快地破坏",
 
-  "rule.portalSuffocationFix.extra.0": "0000000",
-  "rule.portalSuffocationFix.extra.1": "1111111",
+  "rule.movableBlockEntities.name": "容器移动",
+  "rule.movableBlockEntities.desc": "活塞可推动方块实体，比如箱子，漏斗",
+
+  "rule.onePlayerSleeping.name": "单个玩家睡觉",
+  "rule.onePlayerSleeping.desc": "服务器里一个玩家睡觉可跳过夜晚",
+
+  "rule.optimizedTNT.name": "TNT优化",
+  "rule.optimizedTNT.desc": "tnt在相同的地点或在流体里，爆炸造成更小的延迟",
+
+  "rule.persistentParrots.name": "鹦鹉停留",
+  "rule.persistentParrots.desc": "鹦鹉不会离开你的肩膀，除非你收到伤害",
+
+  "rule.placementRotationFix.name": "放置旋转修复",
+  "rule.placementRotationFix.desc": "修复了玩家放置方块时快速转身造成的问题",
+  
+  "rule.pushLimit.name": "自定义推动上限",
+  "rule.pushLimit.desc": "自定义活塞推动上限",
+  "rule.pushLimit.extra.0": "你必须在1-1024中选取一个值",
+  
+  "rule.quasiConnectivity.name": "qC激活开关",
+  "rule.quasiConnectivity.desc": "活塞，发射器和投掷器在他们上方方块激活时是否响应",
+  "rule.quasiConnectivity.extra.0": "设置为true为开启 设置为false为禁用",
+  
+  "rule.railPowerLimit.name": "自定义动力铁轨激活距离",
+  "rule.railPowerLimit.desc": "动力铁轨激活上限可自定义",
+  "rule.railPowerLimit.extra.0": "你必须在1-1024中选取一个值",
+  
+  "rule.renewableCoral.name": "珊瑚可再生",
+  "rule.renewableCoral.desc": "使用骨粉可催熟珊瑚以生成珊瑚结构",
+  
+  "rule.renewableSponges.name": "海绵可再生",
+  "rule.renewableSponges.desc": "守卫者将在被雷击后会变为远古守卫者",
+  
+  "rule.rotatorBlock.name": "仙人掌扳手发射器版",
+  "rule.rotatorBlock.desc": "发射器中的仙人掌可以旋转方块（尽可能以逆时针）",
+  
+  "rule.scriptsAutoload.name": "Scarpet脚本自动加载",
+  "rule.scriptsAutoload.desc": "在服务器/世界加载的时候将会自动从世界文件中加载Scarpet脚本文件",
+  "rule.scriptsAutoload.extra.0": "/script命令必须启用",
+  
+  "rule.shulkerSpawningInEndCities.name": "潜影贝重生",
+  "rule.shulkerSpawningInEndCities.desc": "潜影贝将会在末地城中重生",
+  
+  "rule.silverFishDropGravel.name": "蠹虫钻出方块掉落沙砾",
+  "rule.silverFishDropGravel.desc": "蠹虫钻出方块时会掉落沙砾",
+  
+  "rule.smoothClientAnimations.name": "平滑客户端动画",
+  "rule.smoothClientAnimations.desc": "低TPS设置时可以使客户端动画更加平滑",
+  "rule.smoothClientAnimations.extra.0": "仅在单人游戏中生效，并且会降低客户端运行速度",
+  
+  "rule.stackableShulkerBoxes.name": "潜影盒堆叠",
+  "rule.stackableShulkerBoxes.desc": "空潜影盒仍在地上时将自动堆叠，直至堆叠为1组",
+  "rule.stackableShulkerBoxes.extra.0": "在物品GUI中，请使用shift键+单击以移动整组潜影盒",
+  
+  "rule.summonNaturalLightning.name": "召唤闪电自然化",
+  "rule.summonNaturalLightning.desc": "召唤的闪电将拥有所有自然生成闪电拥有的特性",
+  
+  "rule.superSecretSetting.name": "调试使用",
```

Comment:
Why are you spilling the beans what superSecretSetting does?

Reply:
ok i delected it

Reply:
From what I can tell, superSecretSetting is a master disable, just in case.

It was worrying me at first, as it sounded like a backdoor.